src/app/workflow/launch/launch.component.html

```diff
@@ -25,8 +25,11 @@
       </mat-card>
     </div>
     <div *ngIf="_selectedVersion.valid">
-      <div>
-        <mat-card *ngIf="(isNFL$ | async) === false" class="m-2" matTooltip="Commands for creating a runtime JSON template">
+      <div fxLayout="column" fxLayoutGap="1rem">
+        <mat-card *ngIf="entryType === EntryType.Tool" class="alert alert-info" role="alert">
+          <mat-icon>info</mat-icon> GitHub App Tools are actually launched in workflow mode with the Dockstore CLI
```

Comment:
I'm personally a big user/abuser of "actually", but I think it would be better to not have the word here.